dockerfiles/dev-env-base/.bash_history

```diff
@@ -0,0 +1,2 @@
+ls
```

Comment:
No it is not necessary.
The only necessary is the Dockerfile in the two subfolder and the docker-exec.